pyaerocom/io/test/test_read_mscw_ctm.py

```diff
@@ -0,0 +1,96 @@
+import pytest
+import os
+from pyaerocom.conftest import lustre_unavail, testdata_unavail
+from pyaerocom.io import ReadMscwCtm
+from pyaerocom.io.read_mscw_ctm import ts_type_from_filename
+from pyaerocom.griddeddata import GriddedData
+from pyaerocom.colocation import colocate_gridded_gridded
+from pyaerocom.colocation import ColocatedData
+
+
+def test_read_emep():
+    r = ReadMscwCtm()
+    assert r.data_id == None
+    assert r.vars_provided == []
+    assert r.filepath == None
+    assert r.years_avail == []
+    with pytest.raises(FileNotFoundError):
+        ReadMscwCtm(data_dir='/invalid')
+    with pytest.raises(FileNotFoundError):
+        ReadMscwCtm(filepath='/invalid/test.nc')
+
+
+@testdata_unavail
+def test_read_emep_read_var(path_emep):
+    with pytest.raises(ValueError):
+        r = ReadMscwCtm()
+        r.read_var('od550aer')
+    with pytest.raises(ValueError):
+        r = ReadMscwCtm()
+        r.read_var('od550aer')
+    data_dir = path_emep['data_dir']
+    with pytest.raises(FileNotFoundError):
+        r = ReadMscwCtm(data_dir=data_dir)
+        r.read_var('od550aer')
```

Comment:
Why is this raising FileNotFoundError? Shouldn't the reading work with data_dir specified?

Reply:
Because read_var needs a ts_type to identify which file to use when data_dir is set.
Maybe better to handle this in read_var like:
```
        if self.filepath is None and self.data_dir is None:
            raise ValueError('filepath or data_dir must be set before reading.')
        elif self.data_dir is not None and ts_type is None:
            raise ValueError('ts_type must be specified when reading from directory)
```

Reply:
Updated the tests a bit and changed `read_var` as suggested above.